articles/api/authentication/_logout.md

```diff
@@ -1,16 +1,21 @@
 # Logout
 
```

Comment:
Okay. `oidc/logout` is taking the place of `v2/logout` but is there going to be another entry for `/v2/` logout? We use `v2` for federated logout now, right?